web/packages/quick-editor/src/quick-editor.ts

```diff
@@ -0,0 +1,43 @@
+import { GravatarQuickEditorCore, ProfileUpdatedType, Scope } from './quick-editor-core';
+
+export type QuickEditorOptions = {
+	email: string;
+	editorTriggerSelector: string;
+	avatarSelector: string;
+	scope?: Scope;
+	local?: string;
+}
+
+export default class GravatarQuickEditor {
+	_quickEditor: GravatarQuickEditorCore;
+	_editorTrigger: HTMLElement;
+	_avatarList: NodeListOf<HTMLImageElement>;
+
+	constructor( { email, editorTriggerSelector, avatarSelector, scope, local }: QuickEditorOptions ) {
```

Comment:
Q: Should the  `editorTriggerSelector` be a must parameter to use the library?

```suggestion
	constructor( editorTriggerSelector, { email, avatarSelector, scope, local }: QuickEditorOptions ) {
```

Reply:
The TypeScript definition makes it mandatory. It wouldn't trigger an error in a non-TS environment, but IMO, it feels cleaner to have everything as an object property.
It also keeps both `GravatarQuickEditor` and `GravatarQuickEditorCore` initialization similar.

Reply:
OK, I think [a required option](https://github.com/Automattic/gravatar/pull/14/files/38b6d671a01858bb0952c2a42bf340f1e62f3236#diff-ba766ac9a32a6d44162134bd2dab092d092e8789ba574e3150b4adf4fc22781dR5) should be fine.

Q: I noticed that the [avatarSelector](https://github.com/Automattic/gravatar/pull/14/files/38b6d671a01858bb0952c2a42bf340f1e62f3236#diff-ba766ac9a32a6d44162134bd2dab092d092e8789ba574e3150b4adf4fc22781dR6) option is also required. Do developers need to set it to use the library?

Reply:
No, the `avatarSelector` isn't required. Good catch!